app/assets/stylesheets/govuk_publishing_components/components/_layout-for-public.scss

```diff
@@ -11,3 +11,55 @@
   margin-top: - govuk-spacing(2);
   position: relative;
 }
+
```

Comment:
Would these extra styles that are only for the account things be better placed in their own stylesheet? That way it wouldn't be loaded for applications that don't currently need it.

Reply:
I'm in two minds about this suggestion because on the one hand it sounds like a good idea to optimise the styles like this. But on the other hand, the only place `layout-for-public` component styles seem to be imported currently is [in the application stylesheet in static](https://github.com/alphagov/static/blob/80e0b13f860ec538ad4c006662b9c6f804d00b2c/app/assets/stylesheets/application.scss#L12). 

So let's say we split out the accounty stuff into a new `layout-for-public-accounty-stuff.scss` stylesheet – I'm not sure how I would conditionally import that stylesheet in `application.scss` only when the account layout template is being used if you know what I mean? Not sure if we have any previous examples where we've already done something like that.

Reply:
That makes complete sense - you're right, it'd be a separate stylesheet that'd be included anyway.